keyboards/xbows/knight/knight.c

```diff
@@ -134,7 +134,15 @@
      4, 4, 4,    4, 4, 4, 4,    4, 4, 4, 1, 1, 1
  } };
 
+ void suspend_power_down_kb(void) {
+     rgb_matrix_set_suspend_state(true);
+     suspend_power_down_user();
+ }
 
+ void suspend_wakeup_init_kb(void) {
+     rgb_matrix_set_suspend_state(false);
+     suspend_wakeup_init_user();
+ }
```

Comment:
These are no longer needed. 
```suggestion
```

Reply:
Hi, I've done some tests, and when you keep these lines of code, the keyboard works fine. My keyboard often woke up automatically when I deleted those lines.@drashna

Reply:
This functionality is added to the core code, which is why it's not needed. 

https://github.com/qmk/qmk_firmware/blob/474921d23187499751f14aee444cd78ae6b05a5a/tmk_core/common/avr/suspend.c#L169-L171

As for it waking automatically, I'm not sure, but that shouldn't be related to this,